server.js

```diff
@@ -61,6 +61,49 @@ const EXTERNAL_RESOURCES = {
   search: Strings.isEmpty(Environment.RESOURCE_URI_SEARCH) ? null : Environment.RESOURCE_URI_SEARCH,
 };
 
+let exploreSlates = [];
+
+const fetchExploreSlates = async () => {
+  if (Environment.IS_PRODUCTION) {
+    exploreSlates = await Data.getSlatesByIds({
+      ids: [
+        //NOTE(tara): slates in prod
+        "d2861ac4-fc41-4c07-8f21-d0bf06be364c",
+        "9c2c458c-d92a-4e81-a4b6-bf6ab4607470",
+        "7f461144-0647-43d7-8294-788b37ae5979",
+        "f72c2594-b8ac-41f6-91e0-b2da6788ae23",
+        "a0d6e2f2-564d-47ed-bf56-13c42634703d",
+        "0ba92c73-92e7-4b00-900e-afae4856c9ea",
+      ],
+    });
+
+    for (let exploreSlate of exploreSlates) {
+      let user = await Data.getUserById({ id: exploreSlate.data.ownerId });
+      exploreSlate.username = user.username;
+    }
+  }
+  // else {
+  //   exploreSlates = await Data.getSlatesByIds({
+  //     ids: [
+  //       //NOTE(tara): slates in localhost for testing
+  //       "857ad84d-7eff-4861-a988-65c84b62fc23",
+  //       "81fa0b39-0e96-4c7f-8587-38468bb67cb3",
+  //       "c4e8dad7-4ba0-4f25-a92a-c73ef5522d29",
+  //       "df05cb1f-2ecf-4872-b111-c4b8493d08f8",
+  //       "435035e6-dee4-4bbf-9521-64c219a527e7",
+  //       "ac907aa3-2fb2-46fd-8eba-ec8ceb87b5eb",
+  //     ],
+  //   });
+
+  //   for (let exploreSlate of exploreSlates) {
+  //     let user = await Data.getUserById({ id: exploreSlate.data.ownerId });
+  //     exploreSlate.username = user.username;
+  //   }
+  // }
+};
+
+fetchExploreSlates();
```

Comment:
This is a clever idea! We'll see how it goes :-)

Reply:
why not `exploreSlates = await fetchExploreSlates()`; and `let exploreSlates = []`

Reply:
hmm I think "await" is only allowed in async function. 
if we put `exploreSlates = await fetchExploreSlates()` on top, it will error right?
unless you mean otherwise?

Reply:
You should do it here then: 

```
  const listenServer = server.listen(Environment.PORT, async (e) => {
    if (e) throw e;
    Websocket.create();

    NodeLogging.log(`started on http://localhost:${Environment.PORT}`);
    
    exploreSlates = await fetchExploreSlates();
  });
  ```